common/src/test/java/tachyon/util/FormatUtilsTest.java

```diff
@@ -22,6 +30,144 @@
 
 public class FormatUtilsTest {
   @Test
+  public void parametersToStringTest() {
+    class TestCase {
```

Comment:
TestCase is a local class and as such cannot be declared private (http://docstore.mik.ua/orelly/java-ent/jnut/ch03_11.htm). Are you suggesting I reorganize the code to make it private? I think the current organization makes sense as each TestCase class only pertains to the method in which it is defined.